README.md

```diff
@@ -3,84 +3,181 @@
 [![Issues][issues-badge]][issues-url]
 [![Gitter][gitter-badge]][gitter-url]
 
-Config service Overview
-=======================
+# Config Service Overview
+
 This service handles communication with the [Storage Adapter] microservice to complete tasks.
 
-The microservice provides a RESTful endpoint to make CRUD operations for "devicegroups", "solution-settings", and "user-settings". The data will be stored by the [Storage Adapter] microservice.
+The microservice provides a RESTful endpoint to make CRUD operations for
+"devicegroups", "solution-settings", and "user-settings".
+The data will be stored by the [Storage Adapter] microservice.
+
+## Why?
+
+This microservice was built as part of the 
+[Azure IoT Remote Monitoring](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet)
+project to provide a generic implementation for an end-to-end IoT solution.
+More information [here][rm-arch-url].
+
+## Features
+* Create or update device groups
+* Get all or a single device group
+* Get or upload logo
+* Set or get overall solution settings
+* Set or get individual user settings
+
+## Documentation
+* View the API documentation in the [Wiki](https://github.com/Azure/pcs-config-dotnet/wiki)
+
+# How to Use
+
+## Running the Service with Docker
+You can run the microservice and its dependencies using
+[Docker](https://www.docker.com/) with the instructions
+[here][run-with-docker-url].
+
+## Running the Service Locally
+## Prerequisites
+
+### 1. Deploy Azure Services
+
+This service has a dependency on the following Azure resources. 
+Follow the instructions for 
+[Deploy the Azure services](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-local#deploy-the-azure-services).
+
+* Cosmos DB
+* Iot Hub
+* Maps (optional)
 
-Dependencies
-============
-- [Storage adapter microservice] used to store data
+### 2. Setup Dependencies
 
-How to use the microservice
-===========================
-## Local Setup
+This service depends on the following repositories.
+Run those services from the instructions in their READMEs in the following order.
 
-### 1. Environment Variables
+1. [Storage Adapter Dotnet Microservice](https://github.com/Azure/pcs-storage-adapter-dotnet)
+1. [Telemetry Dotnet Microservice](https://github.com/Azure/device-telemetry-dotnet)
+1. [IoTHub Manager Dotnet Microservice](https://github.com/Azure/iothub-manager-dotnet)
+1. [Device Simulation Dotnet Microservice](https://github.com/Azure/device-simulation-dotnet)
```

Comment:
Can you open a tracking bug for removing this section once the dependencies have been removed.

Reply:
https://github.com/Azure/pcs-config-dotnet/issues/90